arangod/RocksDBEngine/RocksDBEngine.h

```diff
@@ -458,6 +458,23 @@ class RocksDBEngine final : public StorageEngine {
   }
 #endif
 
+  class RocksDBSnapshot : public StorageSnapshot {
+   public:
+    RocksDBSnapshot(rocksdb::DB* db) : _snapshot(db) {}
+
+    TRI_voc_tick_t tick() const noexcept {
+      return static_cast<TRI_voc_tick_t>(
+          _snapshot.snapshot()->GetSequenceNumber());
+    }
+
+    auto getSnapshot() const { return _snapshot.snapshot(); }
```

Comment:
What that gives us?

Reply:
Your function just using for internal field function.

Suggested option makes it clear.

If it will be just auto, if return type changed to reference here will be not intentional copy